src/utl/include/utl/Logger.h

```diff
@@ -88,6 +88,13 @@ namespace utl {
 #define GENERATE_ENUM(ENUM) ENUM,
 #define GENERATE_STRING(STRING) #STRING,
 
+// backward compatibility with fmt versions older than 8
+#if FMT_VERSION >= 80000
+#define FMT_RUNTIME(format_string) fmt::runtime(format_string)
+#else
+#define FMT_RUNTIME(format_string) format_string
+#endif
```

Comment:
Kinda icky but hopefully the compile time checking is beneficial.

Reply:
You only need to add this macro when you are passing a non compile const string, or maybe said another way a dynamic template.

Reply:
Can we add some developer documentation about this requirement in utl? Or could we provide a static function in utl to do this automatically? I worry developer not on c++20 will not know this and break things for you.

Reply:
Can we add some developer documentation about this requirement in utl? Or could we provide a static function in utl to do this automatically? I worry developer not on c++20 will not know this and break things for you.

Reply:
Can we add some developer documentation about this requirement in utl? Or could we provide a static function in utl to do this automatically? I worry developer not on c++20 will not know this and break things for you.

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/docs/contrib/Logger.md would be a good place.  I think the added CI is to prevent such mistakes.

Reply:
Done.